app.vue

```diff
@@ -4,6 +4,8 @@ import type { ParsedContent } from '@nuxt/content/dist/runtime/types';
 const { seo } = useAppConfig();
 
 const { data: navigation } = await useAsyncData('navigation', () => fetchContentNavigation());
+provide('navigation', navigation);
```

Comment:
What does this do?

Reply:
It's a way that makes navigation available across the app: https://nuxt.com/docs/api/composables/use-nuxt-app#provide-name-value

Are we using it actively? Not sure, I'll investigate at a later time if it's actually being used anywhere.